src/caret_analyze/infra/lttng/lttng.py

```diff
@@ -345,13 +435,22 @@ def _parse_lttng_data(
                     events.append(event_dict)
                 filtered_event_count += 1
                 event_name = event[LttngEventFilter.NAME]
-                handler_ = handler.handler_map[event_name]
-                handler_(event)
+                # MYK (memo)
+                # if handler.handler_map[event_name] in None:
+                # None判定だとKeyErrorになる
+                if event_name in handler.handler_map:
+                    handler_ = handler.handler_map[event_name]
+                    handler_(event)
```

Comment:
I suggest to remove if statement because it should be always true.

```suggestion
                handler_ = handler.handler_map[event_name]
                handler_(event)
```

Reply:
I removed the if statement